contents/kotlin/NullSafety.md

```diff
@@ -17,7 +17,7 @@ Author: [Pan Haozhe](https://github.com/Haozhe321)
 >“I call it my billion-dollar mistake. It was the invention of the null reference…My goal was to ensure that all use of references should be absolutely safe, with **checking performed automatically by the compiler**. But I couldn't resist the temptation to put in a null reference, simply because it was so easy to implement.”  
 -Tony Hoare
 
-This document explains Kotlin's null safety feature . For an overview of Kotlin, see [here](https://github.com/se-edu/learningresources/blob/master/contents/kotlin/kotlin.md).
+This document explains Kotlin's null safety feature. For an overview of Kotlin, see [here](https://github.com/se-edu/learningresources/blob/master/contents/kotlin/kotlin.html).
```

Comment:
This one shows a 404 (because it's directly linking to the Github repo and that only has the `.md` file). 

Needs to be replaced with a relative link like you did for `JavaConcurrency` in the C++ Rvalues page.

Reply:
Whoops, thanks for the catch!